packages/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm

```diff
@@ -9,6 +9,11 @@
 #import <React/RCTComponentViewFactory.h> // Allows non-umbrella since it's coming from React-RCTFabric
 #import <ReactCommon/RCTHost.h> // Allows non-umbrella because the header is not inside a clang module
 
+// TODO remove the if when 76 is not supported, or rather remove the EXAppDelegateWrapper because it's deprecated
```

Comment:
```suggestion
// TODO(vonovak,20250122) - Remove the if when 76 is not supported, or rather remove the EXAppDelegateWrapper because it's deprecated
```

James recommended to leave a date on TODO, so people won't forget it